src/components/Dashboard/Modals/EditAppModal/Tabs/AppereanceTab/AppereanceTab.tsx

```diff
@@ -18,16 +19,21 @@ export const AppearanceTab = ({
 }: AppearanceTabProps) => {
   const { t } = useTranslation('layout/modals/add-app');
   const { classes } = useStyles();
+  const { isLoading, error, data } = useQuery({
+    queryKey: ['autocompleteLocale'],
+    queryFn: () => fetch('/api/getLocalImages').then((res) => res.json()),
```

Comment:
Can we maybe use a better API route?
I'd suggest ``/api/images/local``

Reply:
> Can we maybe use a better API route?
> 
> I'd suggest ``/api/images/local``

I don't think it's necessary to create a new folder for one API route, also it only has one method. 

But if we added file uploading... 🤔🤔

Reply:
Just answered this conversation in the following comment:
https://github.com/ajnart/homarr/pull/647#issuecomment-1407472823